core/src/main/java/io/kestra/plugin/core/trigger/Flow.java

```diff
@@ -83,7 +87,7 @@ public class Flow extends AbstractTrigger implements TriggerOutput<Flow.Output>
     @PluginProperty
     private Map<String, Object> inputs;
 
-    public Optional<Execution> evaluate(RunContext runContext, io.kestra.core.models.flows.Flow flow, Execution current) {
+    public Optional<Execution> evaluate(RunContext runContext, io.kestra.core.models.flows.Flow flow, Execution current) throws Exception {
```

Comment:
Evaluating flow trigger is not supposed to fail, so you need to handle the failure and log an exception in the runContext and either skip the flow or don't render.

Another solution would be to create a failed execution but it may be a bit hard.

Reply:
>IllegalVariableEvaluationException is not a runtime exception

Because there is a bunch of places where we must handle all exceptions, for ex in the executor to not crash Kestra and stop. So we use checked exception so we're sure it's handled everywhere.